packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -469,48 +489,170 @@ class HeapTreeViewState extends State<HeapTree>
+                              controller.selectedLeaf = null;
+                            }
+                            setState(() {});
                           }
-                          setState(() {});
                         }
-                      }
-                    : null,
-                child: const Text('Collapse All'),
-              ),
+                      : null,
+                  child: const MaterialIconLabel(
+                    Icons.vertical_align_top,
+                    '',
+                  ),
+                )),
         controller.showTreemap.value
             ? const SizedBox()
-            : OutlineButton(
-                key: expandAllButtonKey,
-                onPressed: snapshotDisplay is MemorySnapshotTable
-                    ? () {
-                        if (snapshotDisplay is MemorySnapshotTable) {
-                          controller.groupByTreeTable.dataRoots
-                              .every((element) {
-                            element.expandCascading();
-                            return true;
-                          });
-                          setState(() {});
+            : Tooltip(
+                message: 'Expand All',
+                child: OutlineButton(
+                  key: expandAllButtonKey,
+                  onPressed: snapshotDisplay is MemorySnapshotTable
+                      ? () {
+                          if (snapshotDisplay is MemorySnapshotTable) {
+                            controller.groupByTreeTable.dataRoots
+                                .every((element) {
+                              element.expandCascading();
+                              return true;
+                            });
+                            setState(() {});
+                          }
                         }
-                      }
-                    : null,
-                child: const Text('Expand All'),
+                      : null,
+                  child: const MaterialIconLabel(
+                    Icons.vertical_align_bottom,
+                    '',
+                  ),
+                ),
               ),
+        const SizedBox(width: defaultSpacing),
+
+        Tooltip(
+          message: 'Monitor Allocations',
+          child: OutlineButton(
+            key: allocationMonitorKey,
+            onPressed: () async {
+              await _allocationStart();
+            },
+            child: createImageIcon(
+              'icons/memory/communities_white@2x.png',
+              size: defaultIconThemeSize,
+            ),
+          ),
+        ),
+        Tooltip(
+          message: 'Reset Accumulators',
+          child: OutlineButton(
+            key: allocationMonitorResetKey,
+            onPressed: () async {
+              await _allocationReset();
+            },
+            child: createImageIcon(
+              'icons/memory/reset_icon_white@2x.png',
```

Comment:
similarly, it would be nice to use a material icon for this as well. We should also show the label rather than a tooltip for these buttons as it isn't obvious enough what they are from the icons.

Reply:
Again added TODO will check with Youyang.